src/main/java/core/basesyntax/Area.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Area {

```

Comment:
this is a bad interface name - way too general, so it's unclear what this interface is used for

how about `AreaCalculator`?